packages/manager/apps/dedicated-servers/src/components/orderMenu.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  ShellContext,
+  useOvhTracking,
+} from '@ovh-ux/manager-react-shell-client';
+import { OdsPopover, OdsButton, OdsLink } from '@ovhcloud/ods-components/react';
+import {
+  ODS_BUTTON_SIZE,
+  ODS_ICON_NAME,
+  ODS_POPOVER_POSITION,
+} from '@ovhcloud/ods-components';
+import useLinkUtils, { UrlLinks } from '@/hooks/useLinkUtils';
+import { orderLinks } from '@/data/constants/orderLinks';
+
+export const OrderMenu: React.FC = () => {
+  const { shell } = React.useContext(ShellContext);
+  const { t } = useTranslation('dedicated-servers');
+  const { trackClick } = useOvhTracking();
+  const links = useLinkUtils<UrlLinks>(orderLinks);
+
+  return (
+    <div className="w-min">
+      <OdsButton
+        icon={ODS_ICON_NAME.plus}
+        size={ODS_BUTTON_SIZE.sm}
+        id={`server-orders-action`}
+        variant="outline"
+        label={t('commander')}
```

Comment:
```suggestion
        id="server-orders-action"
        variant={ODS_BUTTON_VARIANT.outline}
        label={t('commander')}
```

why is the label name in french ? "commander"